assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -13,6 +14,12 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 			toggleSelf: true,
 			hidePrevious: true,
 			autoExpand: true,
+			keyDirection: {
+				ArrowLeft: elementorFrontendConfig.is_rtl ? 1 : -1,
+				ArrowUp: -1,
+				ArrowRight: elementorFrontendConfig.is_rtl ? -1 : 1,
+				ArrowDown: 1,
+			},
```

Comment:
Maybe use `next` and `prev`?

Reply:
These numbers are used for applying direction logic in the entire code

Reply:
Done